R/RunDiagnostics.R

```diff
@@ -88,8 +91,9 @@ runCohortDiagnostics <- function(packageName = NULL,
                                  cohortDatabaseSchema,
                                  vocabularyDatabaseSchema = cdmDatabaseSchema,
                                  cohortTable = "cohort",
+                                 cohortTableNames = CohortGenerator::getCohortTableNames(cohortTable = cohortTable),
```

Comment:
i suggest we add some assertChecks to see if the input values conform to expected